src/api/generatedTypes.ts

```diff
@@ -1353,6 +1353,7 @@ export enum ProspectType {
   RssLogistic = 'RSS_LOGISTIC',
   RssLogisticRecent = 'RSS_LOGISTIC_RECENT',
   SlateScheduler = 'SLATE_SCHEDULER',
+  SlateScheduler_V2 = 'SLATE_SCHEDULER_V2',
```

Comment:
Why are we making a new feed?

Reply:
We want to validate the GCP vs AWS side by side.   The editors don't need to use this feed until we give finish this spike.